From: =?utf-8?q?C=C3=A9dric_Boutillier?= <boutil@debian.org>
Date: Thu, 5 Mar 2020 13:40:48 +0000
Subject: rename the project 'ronn'

Last-Update: 2019-02-06

to provide a drop-in replacement for unmaintained ronn gem
---
 ronn-ng.gemspec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ronn-ng.gemspec b/ronn-ng.gemspec
index d682c85..f01d60d 100644
--- a/ronn-ng.gemspec
+++ b/ronn-ng.gemspec
@@ -1,5 +1,5 @@
 Gem::Specification.new do |s|
-  s.name = 'ronn-ng'
+  s.name = 'ronn'
   s.version = '0.9.0'
   s.date = '2019-12-21'
 
